### DISCUSSION TITLE
addBookmark no effect, 添加标签无效了

### DISCUSSION THREAD

**Discussion by smilechy** [gdh1995/vimium-c#832] [General]
After upgrade, use hotkey to "addBookmark" no longer effect. It shows "Add 1 boobmark", but nothing happened.

**Comment by gdh1995** [comment]
有用啊，```map v addBookmark folder="test"``` 后，就加了这些：
![image](https://user-images.githubusercontent.com/5547703/210123128-56cef152-752d-4781-8671-276986fa51d5.png)

你的浏览器名字、版本号是多少？

**Reply by smilechy** [reply]
Microsoft Edge
版本 108.0.1462.54 (正式版本) (64 位)

**Reply by smilechy** [reply]
Vimium C 1.99.96，Microsoft Edge 108，Windows。
